### PR TITLE
DTSBPS-462-repo

### PR DESCRIPTION
**Pull request by gauravchawla73463**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java [hunk 1]
```diff
@@ -101,6 +101,27 @@ public void findByZipFileName_should_find_envelops_in_db() {
...
+        Envelope e3 = envelope("B.zip", "Z", UPLOAD_FAILURE);
+        e1.setCcdId("1111123");
+        e2.setCcdId("1111123");
+        e3.setCcdId("3546684");
+        dbHas(e1, e2, e3);
+
+        // when
+        List<Envelope> resultA = repo.findByCcdId("1111123");
+        List<Envelope> resultB = repo.findByCcdId("3546684");
+        List<Envelope> resultC = repo.findByCcdId("786473");
```
Comment (mdayican) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java:118]: normally this should be 3 different test case

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java [hunk 2]
```diff
@@ -101,6 +101,27 @@ public void findByZipFileName_should_find_envelops_in_db() {
         assertThat(resultForX).hasSize(0);
     }
 
+    @Test
+    public void findByCcdId_should_find_envelops_in_db() {
```
Comment (mdayican) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java:105]: naming is sth like this should_{expected}_when_{condition}

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java (hunk 1 above)
Comment (gauravchawla73463) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java]: done

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java (hunk 2 above)
Comment (gauravchawla73463) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java]: done

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java [hunk 3]
```diff
@@ -101,6 +101,49 @@ public void findByZipFileName_should_find_envelops_in_db() {
...
+    public void should_find_envelope_when_ccdid_matches() {
+        // given
+        Envelope e1 = envelope("A.zip", "X", Status.CREATED);
+        e1.setCcdId("97586937");
+        dbHas(e1);
+
+        // when
+        List<Envelope> result = repo.findByCcdId("97586937");
+        // then
+        assertThat(result).hasSize(1);
```
Comment (mdayican) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java:114]: do not we need to assert envelope fields?

Comment (gauravchawla73463) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java:115]: Done, was thinking of doing this in serviceTest class

**Review by mdayican — APPROVED** [review]

**Pull request merged**